packages/block-library/src/cover/block.json

```diff
@@ -10,6 +10,10 @@
 		"url": {
 			"type": "string"
 		},
+		"useFeaturedImage": {
```

Comment:
Would it make sense to reduce this to just a `featured` boolean for brevity?

Reply:
@draganescu if we want to keep a prefix we should use `isFeaturedImage` for consistency